README.md

```diff
@@ -241,7 +247,7 @@ $transformer = new Transformer(10, function ($url) use ($browser) {
             return json_decode($response->getBody());
         },
         function (Exception $error) {
-            var_dump('There was an error', $error->getMessage());
+            echo 'Error: ' . $e->getMessage() . PHP_EOL;
```

Comment:
This doesn't look right:

```suggestion
            echo 'Error: ' . $error->getMessage() . PHP_EOL;
```

Or rename `$error` to `$e`?

Reply:
Ah, thank you @clue, I decided that we stay with `$e` for consistency reasons because I used this in every other reject case in this project and in the referenced projects. It should now running correctly.

Reply:
@clue I actually didn't noticed that, nice catch.

@yadaiio Sounds about right, will approve this again :+1: